### PR TITLE
doc: new template

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: doc/source/_templates/navbar.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  <div class="navbar-collapse collapse" id="exCollapsingNavbar2">
+    <a class="navbar-brand py-1" href="{{ pathto(master_doc) }}">
+      <img class="logo" src="{{ pathto('_static/' + logo, 1) }}" alt="{{ project }}"/>
+    </a>
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="https://mergify.io">Mergify.io</a>
+      </li>
+      <li>
+        <a class="nav-link" href="https://dashboard.mergify.io">Dashboard</a>
```
Comment (sileht) [inline doc/source/_templates/navbar.html]: target="_blank"

File: doc/source/_templates/navbar.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  <button class="navbar-toggler ml-auto py-2" type="button" data-toggle="collapse" data-target="#exCollapsingNavbar2">
+    &#9776;
+  </button>
+  <div class="navbar-collapse collapse" id="exCollapsingNavbar2">
+    <a class="navbar-brand py-1" href="{{ pathto(master_doc) }}">
+      <img class="logo" src="{{ pathto('_static/' + logo, 1) }}" alt="{{ project }}"/>
+    </a>
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="https://mergify.io">Mergify.io</a>
```
Comment (sileht) [inline doc/source/_templates/navbar.html]: target="_blank"

File: doc/source/_templates/layout.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+{%- block header %}
+{% include "navbar.html" %}
+{%- endblock %}
+
+{%- block content %}
+  {% include "breadcrumb.html" %}
+  <div class="main container-fluid">
+    <div class="row">
+      <div class="sphinxsidebar col-md-3" aria-label="main navigation">
```
Comment (sileht) [inline doc/source/_templates/layout.html]: I would choice md-2/md-10 instead of md-3/md-9 to have less whitespace between menu and content.

**Comment by jd** [comment]
Fine with me!

**Pull request merged**